deployer/scripts/hawkular.sh

```diff
@@ -196,6 +196,10 @@ EOF
   oc create -f templates/hawkular-cassandra-node-emptydir.yaml
   oc create -f templates/support.yaml
 
+  echo "Creating Hawkular Metrics & Cassandra Config Maps"
+  oc create configmap hawkular-metrics-configuration --from-file=config/hawkular-metrics.properties
+  oc label configmaps hawkular-metrics-configuration metrics-infra=hawkular-metrics
```

Comment:
Similar changes might be required here: https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_hosted_metrics/tasks/install.yml#L76

There *might* be a helper object for config maps, like there is for cluster roles (see https://github.com/jpkrohling/openshift-ansible/blob/d49555f6fcd8810338e01646bd69bd7cf9df649f/roles/openshift_metrics/tasks/install_hawkular.yaml#L25 ).